lib/query-builder.js

```diff
@@ -9,425 +9,442 @@ var operatorMap = {
   'and$': 'AND'
 }
 
-function parseExpression (entp, sTypes, current_name, current_value, params, values) {
-  var result = {
-    processed: true
-  }
+module.exports = function (opts) {
+  // var fromColumn = opts.fromColumnName ? opts.fromColumnName : fromColumnName
```

Comment:
I don't see fromColumn being used. I'd remove it (all together) to avoid misunderstandings.